### DISCUSSION TITLE
demand setting not available FDQY/X50A protocol

### DISCUSSION THREAD

**Discussion by James4L** [revk/xESP32-Faikout#581] [General]
Hi,

Just installed the Faikin into my FDQY ducted system (Australia) and it's working well to control basic controls such as target temp, fan speed, mode, power on/off etc. 

I seem to have much more limited information reported back than some of the screenshots / documentation I see which I believe is the S21 protocol. For temp I only report inlet & liquid, with no outdoor temp reported.

I was really hoping to be able to use the demand feature to set the maximum demand level however it doesn't show as an option on my Faikin and sending the MQTT packet to command/Faikin/demand I get:
{
    "ts": "2024-12-18T14:03:54Z",
    "description": "Setting cannot be controlled",
    "prefix": "command",
    "suffix": "demand",
    "payload": "90"
}

I've searched extensively to find information on X50A protocol differences compared to S21 but I haven't been able to answer my question or gain any clarity. Is the lack of demand command a limitation of the X50A protocol or is it a case that the Faikin hasn't been coded to support this on the X50A protocol, or quite possibly my unit doesn't support the feature?

Thanks!

**Comment by lukyjay** [comment]
Do you have a demand response module installed? The Daikin ducted systems in Australia don't come with this module by default, so I assume it might be required before it will function in this software. Only a guess.

https://www.daikin.com.au/faq/what-demand-enabled-response-dred

The modules are about $350~ excl. installation.
